lib/import/brca/providers/salisbury/salisbury_handler.rb

```diff
@@ -232,10 +249,24 @@ def extract_gene_row(genotype, record)
             genotype.add_gene(gene.first)
           end
 
-          def process_variants(genotype, variant)
-            process_cdna_variant(genotype, variant)
-            process_exonic_variant(genotype, variant)
-            process_protein_impact(genotype, variant)
+          def process_multi_vars(genotype_new, variant, genotypes)
+            variants = variant.split(/;|,/)
+            variants.each do |var|
+              genotype_dup = genotype_new.dup
+              gene = var&.scan(BRCA_REGEX)&.flatten&.uniq
+              if gene.present?
+                genotype_dup.add_gene(gene[0])
```

Comment:
Do we need to try and ensure we keep the structure of the database if there happened to be two variants in the same gene written e.g. "BRCA1 c123A>C; BRCA1 c456G>A"? - this would ideally need to create one record in genetic_test_results which two child records in genetic_sequence_variants, rather than two BRCA1 records 

As, discussed this might be a wider problem in the importers

Reply:
Happy with this after investigations into persister functioning